src/accountsdb/snapshots.zig

```diff
@@ -1834,6 +1834,13 @@ pub const StatusCache = struct {
         return readFromFile(allocator, status_cache_file);
     }
 
+    /// opens the status cache using path {dir}/snapshots/status_cache
+    pub fn initFromDir(allocator: std.mem.Allocator, dir: std.fs.Dir) !StatusCache {
+        const status_cache_file = try dir.openFile("snapshots/status_cache", .{});
```

Comment:
thats a great point - i think a general re-thinking of the directory names / variable names needs to be re-thought - `snapshots_dir` is what contains the snapshot files, ie, the .zstd.tar files (and the unpacked values since we unpack to the same directory) -- which is usually 'validator/accounts_db/'-- because of the way agave packs snapshots, directly unpacking leads to another 'snapshots/' folder being created which can cause confusion -- on top of that, we store other folders in the 'accounts_db' directory which is typically where snapshots are stored leading to more confusion

will leave for now but will create a ticket to reference this

Reply:
https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=90745328